src/cli.ts

```diff
@@ -94,6 +94,14 @@ const cli = meow(`
             --test-sources
                 Will refactor an old plugin source structure like 'src/classes' or 'src/java' to a proper Maven-like structure with
                 'src/main/java' and 'src/test/java'
+        
+        e2e --baseUrl <baseUrl> [--browser <browser>] [--plugins <plugins>]
```

Comment:
If `--baseUrl` is an optional parameter then we should write `[--baseUrl <baseUrl>]`